### PR TITLE
Example with listing of technicians

### PR DESCRIPTION
**Pull request by leonardoandrade**

### CODE REVIEW

File: examples/listTechnicians.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  clientConfig.oauthEndpoint = process.env.AUTH_ENDPOINT;
+}
+
+const client = new fsm.CoreAPIClient(clientConfig);
+
+(async () => {
+  // select the data we need regarding technicians. It can be filtered by job title and external resource flag
+  const coreSQL =
+    `SELECT
+    up.id,
```
Comment (GauSim) [inline examples/listTechnicians.js]: can you prettify this query by using `unifiedPerson` as alias ?

File: examples/listTechnicians.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  clientIdentifier: process.env.CLIENT_IDENTIFIER,
+  clientSecret: process.env.CLIENT_SECRET,
+  clientVersion: process.env.CLIENT_VERSION,
+
+  authAccountName: process.env.AUTH_ACCOUNTNAME,
+  authUserName: process.env.AUTH_USERNAME,
+  authPassword: process.env.AUTH_PASSWORD
+};
+
+// for when the autorization endpoint is different than the usual
```
Comment (GauSim) [inline examples/listTechnicians.js]: can we remove this from the example? this is more an internal thing to develop and test - i think for a consumer its just confusing no?  We should go for prod always.

File: examples/listTechnicians.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+/**
```
Comment (GauSim) [inline examples/listTechnicians.js]: can you also maybe rename to queryTechnicians ?

Comment (leonardoandrade) [reply examples/listTechnicians.js]: ok. Should I rename the query.js to queryServiceCall.js?

**Pull request merged**